libs/components/colorpicker/src/lib/modules/colorpicker/colorpicker.component.ts

```diff
@@ -76,7 +76,7 @@ export class SkyColorpickerComponent implements OnInit, OnDestroy {
   public label: string | undefined;
 
   /**
-   * The HTML element ID (without the leading `#`) of the element that labels the
+   * The HTML element ID of the element that labels the
```

Comment:
It should not have been there to begin with because IDs don't have leading pound signs - that is used for querying but doesn't really make sense to call out in these cases. I had removed the ones for modal last week based on AB#2400048 but hadn't removed the rest. I removed those when doing cleanup yesterday.